### ISSUE TITLE
local variable 'orig_ast_parse' referenced before assignment

### ISSUE THREAD

**Issue by unlimblue** [jbms/sphinx-immaterial#195]
```bash
sphinx_immaterial/apidoc/python/type_annotation_transforms.py", line 245, in _parse_annotation
    sphinx.domains.python.ast_parse = orig_ast_parse
UnboundLocalError: local variable 'orig_ast_parse' referenced before assignment
```

**Comment by 2bndy5** [comment]
can you provide steps to reproduce?

**Comment by jbms** [comment]
I think this occurs with Sphinx 6 --- I created a PR to fix this, but unfortunately it requires some revision still:

https://github.com/jbms/sphinx-immaterial/pull/181

**Comment by jbms** [comment]
Fixed by #181.
